docs/content/guide/storage.md

```diff
@@ -1,69 +1,30 @@
-# Storage
+## Storage
 
 This module is essentially a file manager on your Spica instance. You can store, list and delete your files via this module.
 
-## Listing Storages
+### Online Image Editor
 
-#### With Client
+To edit image typed storage item, click the `three dots` next to the item to open the context menu. Clicking the `edit` button will takes you to the Image Editing page.
 
-You can find the list of your Storage by clicking on the **Storage** from the side menu or by going to the `/storage` URL. If you've just initialized the Spica, the list would be empty.
+On that page you crop the image, scale by percentage and rotate the image as you wish. Click on the `tick` icon to save the image after editing.
 
-#### With API
+### Google Cloud Storage Integaration
 
-Request URL:
+Spica supports Google Cloud Storage out-of-the-box.
 
-> GET "{baseUrl}/storage/{params}"
+First you have to download your service account. To download follow these steps:
 
-Params:  
-All the params below are optional.
+- In the Cloud Console, go to the Service Accounts page.
+- Click Select a project, choose a project, and click Open.
+- Find the row of the service account that you want to create a key for. In that row, click the More more_vert button, and then click Create key.
+- Select the Key type as JSON and click Create.
 
-`"limit"`: Accepts number value. Limits the data count on listing array.
+Clicking Create downloads a service account key file. Make sure to store this file securely, because it can be used to authenticate as your service account.
```

Comment:
```suggestion
Clicking to the Create button, downloads a service account key file. Make sure that you store this file securely because it can be used to authenticate as your service account.
```